### DISCUSSION TITLE
Installation Error: Nefarious BthPS3 Drivers Setup Wizard ended prematurely

### DISCUSSION THREAD

**Discussion by DanielCobalt** [nefarius/BthPS3#43] [Support]
Hello, 

I'm trying to install the BthPS3 Driver as an upgrade from Scptoolkit to get bluetooth working using a genuine PS3 controller. However whenever I attempt to open the .msi file I'm greeted with the following: 
![Capture](https://user-images.githubusercontent.com/20498446/167199279-ac4b7d79-fda3-435d-9807-b679e9f117d9.PNG)

I'm running Windows 10 x64.

Any help would be appreciated.

**Comment by farazdionakbar** [comment]
Same problem here. Got it after uninstalling the 1.x.x version. Pls help

**Comment by SamMal99** [comment]
Had the same Issue. Open `C:\Windows\INF\setupapi.dev`. In that file search for where it says "<<<  [Exit status: FAILURE]". Try to remedy the problem by reading the error. I had to change from x86 to x64 install
